src/electron/CompositeWindow.ts

```diff
@@ -30,12 +31,14 @@ export class CompositeWindow extends BrowserWindow {
 
     // notify react side
     this.webContents.send(IPCMessageNames.ToggleEditMode, this.isInteractive)
+    log('toggled compositor interactivity')
```

Comment:
Throughout the previous code you are using `log.info` or `log.error`; why nothing here?

Reply:
good catch